### ISSUE TITLE
brew cask install cakebrew fails due to SSL certificate problem 

### ISSUE THREAD

**Issue by lukaleli** [brunophilipe/Cakebrew#187]
Apparently SSL certificate is no longer valid for https://cdn.cakebrew.com domain, thus installing cakebrew via homebrew fails.

**Comment by brunophilipe** [comment]
The cask reference should be updated as I no longer posses a certificate for cdn.cakebrew.com. The following URL should be used instead:

https://cakebrew-377a.kxcdn.com/cakebrew-1.2.3.dmg

**Comment by srustamo** [comment]
Any chance publishing shasum on the website for the dmg, barring a quick cask ref update?

**Comment by brunophilipe** [comment]
I have added a SHA256 hash to every file in both the CDN and the cakebrew website. They all follow the file URL with an extra `.sha256` extension, such as:

`https://www.cakebrew.com/files/cakebrew-1.2.3.dmg.sha256`
`https://cakebrew-377a.kxcdn.com/cakebrew-1.2.3.dmg.sha256`

The only problem is that the CDN doesn't allow me to set the mime type for the sha256 extension so that it doesn't force a download, but I guess this is probably not a problem for an automated tool? In any case, I was able to do that in `cakebrew.com`, so it should work properly there, at least.
